docs/containers-storage.conf.5.md

```diff
@@ -93,10 +93,10 @@ until all of the entries have been used for maps.
       remap-user = "storage"
       remap-group = "storage"
 
-ostree_repo = ""
-   If specified, use OSTree to deduplicate files with the overlay backend.
+**ostree_repo** = ""
+   If specified, use OSTree to deduplicate files with the overlay or vfs backends.
 
-skip_mount_home = "false"
```

Comment:
Let's keep the default value. Currently, it looks like a string but only "true" or "false" are valid values.

Reply:
> Let's keep the default value. Currently, it looks like a string but only "true" or "false" are valid values.

Done